### ISSUE TITLE
PR explanation

### ISSUE THREAD

**Issue by OffifialMITX** [LonamiWebs/Stringlate#227]
It would be nice if SL had an explanation about PRs im the help section.

Like, I am a noob and I don't really know what it does, but people keep suggesting I should use them, but when I try, it says it will clone the repo, which does not sound like something I want, but I have no idea.

**Comment by licaon-kter** [comment]
Yes. You fork the repo actually (Github does) and you commit your modifications to your repo, then you open a PR from your repo with these changes to the original repo for integration in there.

Read about how Github works.

**Comment by OffifialMITX** [comment]
That's what I thought, but I wasn't sure. It wouldnbe nice if you influded that text (more or less) in the current help.

On June 5, 2018 7:58:16 AM GMT+02:00, Licaon_Kter <notifications@github.com> wrote:
>Yes. You fork the repo actually (Github does) and you commit your
>modifications to your repo, then you open a PR from your repo with
>these changes to the original repo for integration in there.
>
>Read about how Github works.
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/LonamiWebs/Stringlate/issues/227#issuecomment-394591045

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by OffifialMITX** [comment]
Which is the point of this issue.
-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by OffifialMITX** [comment]
Sorry for noobing around, just wanted to be sure.
-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.
